CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ endif (CMAKE_VERSION VERSION_LESS 3.9)
 # fix macOS RPATH issues
 set (CMAKE_MACOSX_RPATH 1)
 set (CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
+set (CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
```

Comment:
Should we really add the rpath to **all** installed binaries? I think rpath is generally discouraged for packaging IIRC.
If the rpath however is only needed for jni/jna, I think it would be best to only add it for the jni/jna target, like in https://github.com/ElektraInitiative/libelektra/pull/3755.
@kodebach maybe this was also the reason why the packaged kdb tests failed? (only a guess)

Reply:
@kodebach i will wait for your response regarding this issue...
If we are still out of ideas, i will reinstate the DeletLocalRef in jni.c and comment out kdb all in Jenkinsfile, so we can proceed like discussed after the Elektra meeting...
Since the sporadic testjna_gradle segfault most likely is associated with the Cleaner...

Reply:
Yes, this could be the reason, although I really don't know so just try and see what works...

Reply:
@robaerd i've tried, but i honestly don't really know what i'am doing ;)
could you maybe give me some insight or just try it yourself / propose a change?